src/dist/replication/test/meta_test/unit_test/meta_duplication_service_test.cpp

```diff
@@ -678,8 +694,56 @@ TEST_F(meta_duplication_service_test, query_duplication_handler)
     ASSERT_EQ(fake_resp.body,
               std::string() + R"({"1":{"create_ts":")" + ts_buf + R"(","dupid":)" +
                   std::to_string(dup->id) +
+                  R"(,"fail_mode":"FAIL_SLOW")"
                   R"(,"remote":"slave-cluster","status":"DS_START"},"appid":2})");
 }
 
+TEST_F(meta_duplication_service_test, fail_mode)
+{
+    std::string test_app = "test-app";
```

Comment:
I rember we don't support the charactor of `-`

Reply:
'-' is not restrictively forbidden in Pegasus. So creating such a table in unit test is OK.